infinispan-spring-boot-starter-remote/src/main/java/org/infinispan/spring/starter/remote/InfinispanRemoteAutoConfiguration.java

```diff
@@ -64,10 +64,14 @@ public RemoteCacheManager remoteCacheManager() throws IOException {
 
       boolean hasHotRodPropertiesFile = ctx.getResource(infinispanProperties.getClientProperties()).exists();
       boolean hasConfigurer = infinispanRemoteConfigurer != null;
+      boolean hasConfiguration = infinispanConfiguration != null;
       boolean hasProperties = StringUtils.hasText(infinispanProperties.getServerList());
 
       org.infinispan.client.hotrod.configuration.Configuration configuration;
-      if (hasConfigurer) {
+      
+      if (hasConfiguration) {
+         configuration = infinispanConfiguration;
```

Comment:
This change looks fine although it it's not symmetric with `InfinispanEmbeddedAutoConfiguration`. The `InfinispanEmbeddedAutoConfiguration` checks whether an XML file is accessible and then checks other configuration options. I believe this is a correct approach where configuration files are somewhat "stronger" than POJO configuration.

May I ask you to move `configuration = infinispanConfiguration` into if statement below (where it was before)?

Reply:
Alternatively to this, we can grab POJO configuration and merge properties and configurer configurations into it using `ConfigurationBuilder#read()` method but I think this kind of solution is less intuitive.